BiweeklyMeetings/2022-08-02.md

```diff
@@ -0,0 +1,323 @@
+
+**Anthony**-: Amazing. Great.
+
+#### PR296
+[PR296 CIP-0062? | Governance API for dApp Connectors (dApps<->Wallet Web-bridge](https://github.com/cardano-foundation/CIPs/pull/296)
+
+**Matthias** -: CIP 62 is next in line, right?
+
+**Anthony**-: That's correct.
+
+**Matthias** -: I've been trying to click on the link for 15 seconds and I just realized it was a stream. So can click on the video, obviously. Governance API for DApp, which was proposed by [inaudible 00:36:17] Martins, who is now the tech lead regarding Wallets at IOG correctly. And this is a proposal to extend CIP 30 with capabilities that would make that voting process a bit more streamlined. Sorry, I did a first run of [inaudible 00:36:42] on that one this morning and mainly suggested to simplify a bit some of the methods in particular to make things a bit more [inaudible 00:36:54], meaning not to have the API provide five different ways of doing the same things or in that case two different ways.
+
+**Matthias** -: And I'm also asking for a rationale section mainly because we need a rationale section for any CIPs in general, because that what exists the discussion for all the bases for discussion and that explains, at a later time, while the decisions were made, the waiting were made. And for that one in particular, because there are a few suggested API or endpoints, you could say, or interfaces, that can be done currently through existing libraries and I don't see why adding them to the bridge, the wallet interface, so to speak makes sense. Or at least there is no justification for it. I can get that it's maybe a bit more simple and that removes some of the burden from the apps implementing that, but those apps are mainly Wallets anyway. So that was a bit some form of the gap in the justification here that I'm liking. I don't know, Sebastien, if you had a look at that yet.
+
+**Sebastien** -: No, unfortunately I was hoping to look into it today, but as you may have heard, there's other stuff that came up. There's still some discussion going on internally related to the CIP, so stuff may still change, but definitely it's good to over it. I agree that it's probably over complicated the way it's testified now and I'll try and find some time to give my thoughts on this.
+
+**Matthias** -: Okay. Because my main understanding on this, or I mean, experience that's... I mean the catalyst voting in general, it's always been done by [inaudible 00:39:01] themselves. So as part of the Wallet interface, usually do things a bit differently that usually Turmo for that in the past, it's pretty nice just acreage transaction registration as part of the Wallet itself. So I don't quite see the need for an external API for that to let any depth to the registration themselves.
```

Comment:
```suggestion
**Matthias** -: Okay. Because my main understanding on this, or I mean, experience that's... I mean the catalyst voting in general, it's always been done by wallet implementors themselves; [Inaudible] So I don't quite see the need for an external API for that to let any DApp do the registration themselves.
```